quadlet/redpanda/sample-app/admin.py

```diff
@@ -0,0 +1,19 @@
+from kafka import KafkaAdminClient
+from kafka.admin import NewTopic
+from kafka.admin import KafkaAdminClient, NewTopic
+
+class Admin:
+    def __init__(self, brokers):
```

Comment:
Please add some empty lines between methods

Reply:
[5110298](https://github.com/containers/appstore/pull/14/commits/5110298a0e1738d9d2a7992f9924ad81bbf02535)

Reply:
I think that inside a class it's one empty line  (two lines at the main level).

Reply:
Thanks.  Ran it through a formatter and you're right.  Inside a class, it's a single line.

cc54120